tests/test_fmu_dataio_table.py

```diff
@@ -30,30 +31,55 @@
 CASEPATH = "tests/data/drogon/ertrun1"
 
 
-def test_table_io(tmp_path):
+def test_table_io_pandas(tmp_path):
     """Minimal test tables io, uses tmp_path."""
 
     # make a small DataFrame
-    table = pd.DataFrame({"STOIIP": [123, 345, 654], "PORO": [0.2, 0.4, 0.3]})
+    df = pd.DataFrame({"STOIIP": [123, 345, 654], "PORO": [0.2, 0.4, 0.3]})
     fmu.dataio.ExportData.export_root = tmp_path.resolve()
     fmu.dataio.ExportData.table_fformat = "csv"
 
     exp = fmu.dataio.ExportData(name="test", verbosity="INFO", content="volumes")
     exp._pwd = tmp_path
-    exp.to_file(table)
+    exp.to_file(df)
 
     assert (tmp_path / "tables" / ".test.csv.yml").is_file() is True
     with open(tmp_path / "tables" / "test.csv") as stream:
         header = stream.readline().split(",")
     assert len(header) == 2
 
     # export with index=True which will give three columns (first is the index column)
-    exp.to_file(table, index=True)
+    exp.to_file(df, index=True)
     with open(tmp_path / "tables" / "test.csv") as stream:
         header = stream.readline().split(",")
     assert len(header) == 3
 
 
+def test_table_io_arrow(tmp_path):
+    """Test the support for PyArrow tables"""
+
+    # make a small pa.Table
+    df = pd.DataFrame({"STOIIP": [123, 345, 654], "PORO": [0.2, 0.4, 0.3]})
+    table = pa.Table.from_pandas(df)
+    fmu.dataio.ExportData.export_root = tmp_path.resolve()
+
+    exp = fmu.dataio.ExportData(name="test", verbosity="INFO", content="smry")
+    exp._pwd = tmp_path
+    exp.to_file(table)
+
```

Comment:
Instead of setting `exp._pwd`, I think is better to set `ExportData(... runfolder=tmp_path)` since `self._pwd` is actually used in the __init__.

Reply:
Done ✔